validator/client/BUILD.bazel

```diff
@@ -97,10 +96,9 @@ go_library(
 
 go_test(
     name = "go_default_test",
-    size = "small",
+    size = "medium",
```

Comment:
why the change here?

Reply:
Because, with new added tests,  if `small`:
```
//validator/client:go_default_test                                      TIMEOUT in 60.0s
```

Tests are longer to run mainly because some tests now run for both complete, boltDB and minimal, filesystem databases.